### ISSUE TITLE
[🐛 Bug]: google assistant changes microphone volume

### ISSUE THREAD

**Issue by citrinador8797** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#585]
### What went wrong?

the google assistant lowers my microphone volume automatically when the "hey google / ok google" option is activated

### What did you expect to happen?

shouldn't change my microphone volume

### Steps to reproduce the bug

1. enable "hey google/ok google".
2. now just say something that makes a lot of noise.
3. your microphone volume will decrease.

### App Version

v1.0.0

### OS Version

Windows 11 Pro 21H2 build 22000.318

### :information_source: Platform Info
Running on **Windows_NT x64 10.0.22000**

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%F0%9F%90%9B+bug%22) and didn't find any duplicate issue
- [X] I have checked the [Frequently Asked Questions (FAQs)](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)) and can confirm that there's no fixes or workarounds for this issue
- [X] This issue only covers one particular bug or issue

**Comment by Melvin-Abraham** [comment]
@citrinador8797 That seems to be a misconception as this application does not do that. Quite possibly there are other applications changing the microphone volume. I happen to suspect that you might have Zoom running on your system which actually does that by default. Please do check that.

**Comment by Melvin-Abraham** [comment]
Duplicate of #203

**Comment by Melvin-Abraham** [comment]
_Closing as duplicate_

**Comment by citrinador8797** [comment]
but this option doesn't appear to me


Em qui., 18 de nov. de 2021 às 22:59, Melvin. L. Abraham <
***@***.***> escreveu:

**Comment by citrinador8797** [comment]
> Duplicate of #203

but this option doesn't appear to me

**Comment by citrinador8797** [comment]
> Duplicate of #203

and this problem is caused by the app, because after i disabled the option "hey google / ok google" it stopped happening, but this option is very useful, so i need to keep it active

**Comment by im-coder-lg** [comment]
Well, it's fairly simple. GAUDC uses the Bumblebee Hotword and to hear clearly, it reduces the system volume by maybe 2 dB(really significant) so it's just normal, this is how the Android GA responds too, so this is normal(fairly). However, you might have to be careful with meeting apps or apps like Spotify since there's an external issue(tracked here: #472) but since this is external, this can't be fixed here.

**Comment by Melvin-Abraham** [comment]
@citrinador8797 Can you share screenshots and/or a video to demonstrate the issue. Since, I can't reproduce the issue on my system.

**Comment by Melvin-Abraham** [comment]
Hold on, I guess I see the confusion. The Ok Google / Hey Google might not work perfectly as it is implemented by the bumblebee library which in turn uses picovoice. The built-in model for the same isn't as comprehensive as that of the official model by Google thus you might have to say it loudly or in a particular tone for it to get it.

**Comment by im-coder-lg** [comment]
> Hold on, I guess I see the confusion. The Ok Google / Hey Google might not work perfectly as it is implemented by the bumblebee library which in turn uses picovoice. The built-in model for the same isn't as comprehensive as that of the official model by Google thus you might have to say it loudly or in a particular tone for it to get it.

Same, you gotta shout properly (I literally mean it) to get GAUDC up. One might think it's mic issues. But it's Bumblebee. I wonder if you can configure Bumblebee to wake GAUDC when one says `Wake` or something shorter.

**Comment by citrinador8797** [comment]
> @citrinador8797 Can you share screenshots and/or a video to demonstrate the issue. Since, I can't reproduce the issue on my system.

https://drive.google.com/file/d/1KrgtIdm5fj4A2Lqp8r1coF8HY4nVNAn4/view?usp=sharing  the video was bigger than 10 mb so i had to upload it to google drive

**Comment by citrinador8797** [comment]
> Well, it's fairly simple. GAUDC uses the Bumblebee Hotword and to hear clearly, it reduces the system volume by maybe 2 dB(really significant) so it's just normal, this is how the Android GA responds too, so this is normal(fairly). However, you might have to be careful with meeting apps or apps like Spotify since there's an external issue(tracked here: #472) but since this is external, this can't be fixed here.

but it just lowers the microphone volume. Should I turn the volume up again when I say something low, but it doesn't

**Comment by citrinador8797** [comment]
> Hold on, I guess I see the confusion. The Ok Google / Hey Google might not work perfectly as it is implemented by the bumblebee library which in turn uses picovoice. The built-in model for the same isn't as comprehensive as that of the official model by Google thus you might have to say it loudly or in a particular tone for it to get it.

this is not the problem. I even figured it out. The problem is that it lowers the volume on my microphone and then it doesn't increase automatically, that is, the volume is low

**Comment by citrinador8797** [comment]
> > Hold on, I guess I see the confusion. The Ok Google / Hey Google might not work perfectly as it is implemented by the bumblebee library which in turn uses picovoice. The built-in model for the same isn't as comprehensive as that of the official model by Google thus you might have to say it loudly or in a particular tone for it to get it.
> 
> Same, you gotta shout properly (I literally mean it) to get GAUDC up. One might think it's mic issues. But it's Bumblebee. I wonder if you can configure Bumblebee to wake GAUDC when one says `Wake` or something shorter.

Yes, it's not a mic problem, it's a Bumblebee problem. Would it be possible to remove this part of the automatic microphone adjustment?

**Comment by Melvin-Abraham** [comment]
> > Hold on, I guess I see the confusion. The Ok Google / Hey Google might not work perfectly as it is implemented by the bumblebee library which in turn uses picovoice. The built-in model for the same isn't as comprehensive as that of the official model by Google thus you might have to say it loudly or in a particular tone for it to get it.
> 
> this is not the problem. I even figured it out. The problem is that it lowers the volume on my microphone and then it doesn't increase automatically, that is, the volume is low

@citrinador8797 How can you confirm that the mic volume has been decreased? Can you share any screenshot confirming that?

**Comment by citrinador8797** [comment]
> How can you confirm that the mic volume has been decreased? Can you share any screenshot confirming that?

you can see this in the video I uploaded in the comments above. (https://drive.google.com/file/d/1KrgtIdm5fj4A2Lqp8r1coF8HY4nVNAn4/view?usp=sharing)

**Comment by Melvin-Abraham** [comment]
> > How can you confirm that the mic volume has been decreased? Can you share any screenshot confirming that?
> 
> you can see this in the video I uploaded in the comments above. (https://drive.google.com/file/d/1KrgtIdm5fj4A2Lqp8r1coF8HY4nVNAn4/view?usp=sharing)

The video is restricted it seems. You may set the access to public (to confirm try opening the link in an incognito window) or attach the video file by dragging and dropping the it directly in the comment box.

**Comment by citrinador8797** [comment]
oh sorry, i fix it
https://drive.google.com/file/d/1KrgtIdm5fj4A2Lqp8r1coF8HY4nVNAn4/view?usp=sharing

Em sex., 19 de nov. de 2021 às 11:11, Melvin. L. Abraham <
***@***.***> escreveu:
